app/models/user.rb

```diff
@@ -0,0 +1,19 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+
+  validates_presence_of :name
+
+  has_many :posts, dependent: :destroy
+  has_many :friend_requests_sent, foreign_key: :sender_id,
+            class_name: "FriendRequest", dependent: :destroy
+  has_many :friend_requests_received, foreign_key: :receiver_id,
+			class_name: "FriendRequest", dependent: :destroy
+  has_many :comments, dependent: :destroy
+  has_many :friendships, dependent: :destroy
+  has_many :friends, through: :friendships
```

Comment:
Please check out this bug. So what I did was create 2 users `ryto` and `mike`. Initially they're not friends, right?

Ok, so what I did was:

`ryto.friendships.create(friend_id: mike.id)`

After this I would expect `ryto` and `mike` now to be appearing in each other's friends list, right? 

So, I ran `ryto.friends`. Sure enough `mike` is there:

![image](https://user-images.githubusercontent.com/6015315/57743789-11dad880-76f9-11e9-9f0b-b9ad0b4fefe7.png)

However, doing the same for `mike`, running `mike.friends` results in:

![image](https://user-images.githubusercontent.com/6015315/57743817-2d45e380-76f9-11e9-8049-4e03e8217e8d.png)

The friendship didn't reflect on `mike`'s side. Such that mike still has no friends.

We want both of them to reflect the friendship. In this case, you need to do something about your user.